main.py

```diff
@@ -0,0 +1,71 @@
+import os
+from scripts.dna_tool import dna_rna_tool
+from scripts.protein_tool import protein_tool
+from scripts.fastq_tool import fastq_tool
+
+
+def transform_file_to_dict(input_path: str) -> dict:
```

Comment:
Мы ж не просто любой файл берем. Да и что значит трансформировать? 
```suggestion
def read_fastq(input_path: str) -> dict:
```

:)